nominees/reveal.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Reveal",
+  "aliases": [
+    ""
+  ],
+  "description": "Reveal is an open-source platform that enables country governments and implementing partners to use geospatial data and technology to efficiently and effectively deliver life-saving interventions.",
+  "website": "https://revealprecision.com",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/revealprecision"
```

Comment:
This link points to a repository not a license, and could not find a license in any of the repositories of that organization